modules/migrating-to-multi-arch-cli.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * updating/migrating-to-multi-payload.adoc
+
+:_content-type: PROCEDURE
+[id="migrating-to-multi-arch-cli_{context}"]
+= Migrating a cluster to a heterogenous payload using the CLI
+
+.Prerequisites
+
+* You must be logged in as a user with `cluster-admin` privileges on all clusters.
+* You must be updated to {product-title} version 4.13.0 or higher. For more information on how to update your cluster version, see _Updating a cluster using the web console_ or _Updating a cluster using the CLI_ listed in the Additional resources section.
+
+.Procedure
+
+. Install your {product-title} cluster on your preferred platform. For more information on selecting a supported platform for your cluster installation, see _Selecting a cluster installation type_ in the Additional resources section.
+. Set the channel to `stable-4.13`, `fast-4.13`, or `eus-4.13` with `oc adm upgrade channel <channel>`. For more information about channels, refer to _Understanding update channels and releases_ listed in the Additional resources section.
```

Comment:
> `eus-4.13`
We will not have eus-4.13 channel. I guess you intend to say candidate-4.13 :)

Reply:
We should make sure the current cluster version is included in the channel, or else it's impossible to get the mutli-arch payload to migrate.

Reply:
The EUS channels are only created for EUS versions. So there will not be eus-4.13 channel.

Reply:
We should remove `4.13` from `stable-4.13`, `fast-4.13`, or `eus-4.13` so that it will be generic.

Reply:
sounds good! so - just stable and fast?

Reply:
I can use the same generic terms that I used in the EUS docs (ie stable-<4.y>)